test/e2e/framework/test_context.go

```diff
@@ -103,6 +103,8 @@ type TestContextType struct {
 	// Node e2e specific test context
 	NodeTestContextType
 
+	// Monitoring solution that is used in current cluster. Values are coming from ENABLE_CLUSTER_MONITORING flag.
+	ClusterMonitoring string
```

Comment:
Add a newline before the next line (like it was before). And remove the one above - it's not needed imo.